nlu.py

```diff
@@ -20,6 +20,38 @@ def _get_commands(text_sequence: [str]) -> set:
     return possible_commands
 
 
+def _consine_similarity(a: [str], b: [str]) -> float:
+    words = set(a+b)
+    vector_a = _word2vector(words, a)
+    vector_b = _word2vector(words, b)
+    return _cosine_theta(vector_a, vector_b)
+
+
+def _cosine_theta(a: [int], b: [int]):
+    numerator = sum(x1 * x2 for x1, x2 in zip(a, b))
```

Comment:
So check the length of `a`, `b`, it should say something wrong with different length, do not eat the error. No rely on others component.

Reply:
sure.